CIFAR10/inference/BAIDU-USA-GAIT_ResNet18_1GTX1080_pytorch.json

```diff
@@ -0,0 +1,17 @@
+{
+	    "version": "v1.0",
+	    "author": Baidu USA GAIT LEOPARD team: Baopu Li, Jiazhuo Wang, Zhiyu Cheng, Haofeng Kou, Yingze Bao 

```

Comment:
This is not valid JSON. Please fix and make sure the unit tests pass. From the root directory of this repo, you can run the unittests locally with the following:

```
pip install -r requirements.txt
pytest
```